### PR TITLE
S7 NN update

### LINKED ISSUE #37: dPoW Hardfork update
The KMD hardfork update for season 7 has been released :tada: and is scheduled for the 30th of June.
Your will need to update the Marmara code to include the new pubkeys and calculate an estimated MCL block height that corresponds to close to the same date / time.

You can reference this pull request to get the required changes https://github.com/KomodoPlatform/AYAv2/pull/5

### PR DESCRIPTION
**Pull request by Aklix**
fixes #37

### CODE REVIEW

**Review by DeckerSU — DISMISSED** [review]
LGTM. The only thing, seems you have versioning mismatch. If latest version of software was `1.1.7` according to tags / latest releases, then this version should be `1.1.8` and `configure.ac` and `clientversion.h` should contain corresponding version values:
```cpp
define(_CLIENT_VERSION_MAJOR, 1)
define(_CLIENT_VERSION_MINOR, 1)
define(_CLIENT_VERSION_REVISION, 8)
```
and 
```
#define CLIENT_VERSION_MAJOR 1
#define CLIENT_VERSION_MINOR 1
#define CLIENT_VERSION_REVISION 8
```
accordingly.

File: src/version.h [hunk 1]
```diff
@@ -24,7 +24,7 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 170009;
+static const int PROTOCOL_VERSION = 1700010;
```
Comment (DeckerSU) [inline src/version.h]: Also, I guess here we wanted `170010`, not the `1700010`.
```suggestion
static const int PROTOCOL_VERSION = 170010;
```

**Comment by dimxy** [comment]
I see that we also added a deprecation height in the kmd codebase, to shutdown nodes that did not upgrade. Do we not need similar in marmara?

**Comment by DeckerSU** [comment]
> I see that we also added a deprecation height in the kmd codebase, to shutdown nodes that did not upgrade. Do we not need similar in marmara?

Current MCL block is [1773045](https://explorer.marmara.io/block/0000003ff11036dea51ca428271eaf6892941229ade52bf5a2d88fc4c0fdf4dd), current `DEPRECATION_HEIGHT` is `3600000`. I guess it's too far from current block height, so, we can leave everything as is.

**Comment by Aklix** [comment]
#39   changed versions, what should be `DEPRECATION_HEIGHT` ?

**Comment by dimxy** [comment]
> #39 changed versions, what should be `DEPRECATION_HEIGHT` ?

As I can see the deprecation code is supported for the KMD net only, so we should not do anything for that.
Let's stick with the protocol version

**Review by DeckerSU — APPROVED** [review]
LGTM now. Btw, all the changes merged / cherry-picked to KomodoOcean branch of Qt wallet also, so, everybody who wants to build Marmara Qt can do it from - https://github.com/DeckerSU/KomodoOcean/tree/static-mcl .

**Review by dimxy — APPROVED** [review]
LGTM too

**Pull request merged**